.readme-partials.yaml

```diff
@@ -0,0 +1,8 @@
+custom_content: |
+  Cloud Firestore is a fully-managed NoSQL document database for mobile, web,
+  and server development from Firebase and Google Cloud Platform.  It's backed
+  by a multi-region replicated database that ensures once data is committed,
+  it's durable even in the face of unexpected disasters. Not only that,
+  but despite being a distributed database, it's also strongly consistent and
+  offers seamless integration with other Firebase and Google Cloud Platform
+  products, including Google Cloud Functions.
```

Comment:
You may want to put this in the .repo-metadata.json in the `api_description` field starting with "is a fully-managed..."

Reply:
Done